src/Shared/E2ETesting/BrowserTestBase.cs

```diff
@@ -32,14 +32,16 @@ public BrowserTestBase(BrowserFixture browserFixture, ITestOutputHelper output)
 
         public BrowserFixture BrowserFixture { get; }
 
+        public abstract string BrowserIsolationContext { get; }
+
         public Task DisposeAsync()
         {
             return Task.CompletedTask;
         }
 
         public virtual async Task InitializeAsync()
         {
-            var (browser, logs) = await BrowserFixture.GetOrCreateBrowserAsync(Output);
+            var (browser, logs) = await BrowserFixture.GetOrCreateBrowserAsync(Output, BrowserIsolationContext);
```

Comment:
This is only needed in a couple of tests where we want to ensure a clean slate within the tests. Please undo this change